test-app/schema/src/main/kotlin/com/example/redwood/testing/schema.kt

```diff
@@ -95,6 +96,12 @@ public data class TextInput(
   @Property(5) val maxLength: Int,
 )
 
+@Widget(8)
+public data class Rectangle(
+  @Property(1) val backgroundColor: UInt?,
+  @Property(2) val cornerRadius: Float?,
```

Comment:
Currently these are nullable but have no default value and the implementation does not handle null. I would probably make them non-null, and consider a default corner radius of 0f.